main.py

```diff
@@ -0,0 +1,167 @@
+import json
+import datetime
+
+isAlarm = True
+mainTmp = -1
+hideoutTmp = -1
+traderTmp = -1
+optionTmp = -1
+skillLevel = 0
+Maker = []
+
+def saveJsonTime() :
+    f = open('time.json', 'w')
+    json.dump(Maker, f)
+    f.close()
+
+def initStartProgram() : 
+    for i in range(9) :
+        Maker.append(datetime.datetime(2020, 1, 1, 1, 1, 1))
+    
+def printMain() :
+    print("Welcome to Tarkov Timer. Select number \n\
+============================================= \n\
+1. Hideout \n\
+2. Traders \n\
+3. Options \n\
+4. Github \n\
+\n\
+0. Exit")
+
+def printHideout() :
+    print("Select hideout \n\
+1. WorkBench \n\
+2. Medstation \n\
+3. Lavatory \n\
+4. Scav Case \n\
+5. Booze Generator \n\
+6. Water Collector \n\
+7. Nutrition Unit \n\
+8. Intelligence Center \n\
+\n\
+0. Go Main")
+
+def printTraders() :
+    print("Select traders \n\
+1. Prapor \n\
+2. Therapist \n\
+3. Skier \n\
+4. Peacekeeper \n\
+5. Mechanic \n\
+6. Ragman \n\
+7. Jaeger \n\
+\n\
+0. Go Main")
+
+def printOption() :
+    print("Select Options \n\
+1. Alarm \n\
+2. Skill Level \n\
+\n\
+0. Go Main")
+
+def getInputNumber() :
+    strTmp = input('Input : ')
+    if(strTmp.isdigit()) :
+        tmp = int(strTmp)
+        return tmp
+    else :
+        print('Error! - Please input the correct number.')
+        return -1
+
+def funMain(inputNum) :
+    if(inputNum == -1) :
+        printMain()
+        inputNum = getInputNumber()
+    elif(inputNum == 0) :
+        return 0;
+    elif(inputNum == 1) :
+        printHideout()
+        hideoutTmp = getInputNumber()
+        if(hideoutTmp == 0) :
+            inputNum = -1;
+        else :
+            funHideout(hideoutTmp)
+    elif(inputNum == 2) :
+        printTraders()
+        traderTmp = getInputNumber()
+        if(traderTmp == 0) :
+            inputNum = -1;
+        else :
+            funTrader(traderTmp)
+    elif(inputNum == 3) :
+        printOption()
+        optionTmp = getInputNumber()
+        if(optionTmp == 0) :
+            optionTmp = -1;
+        else :
+            funOption(optionTmp)
+    elif(inputNum == 4) :
+        print("Github 주소")
+        inputnum = -1;
+    else :
+        print("Error! - Please input the correct number.")
+        return -1;
+    
+    return inputNum
+
+def funTrader(inputNum) :
+    print("아직 미구현 기능입니다. 다음에 이용해주세요. 감사합니다. \n")
+    return 0;
+
+def funOption(inputNum) :
+    if(inputNum == 0) :
+        return 0;
+        
+    elif(inputNum == 1) :
+        if(isAlarm) :
+            isAlarm = False
+            print("End Alarm")
+        else :
+            isAlarm = True
+            print("Start Alarm")
```

Comment:
```suggestion
        isAlarm = not isAlarm
        print('End Alarm' if isAlarm not 'Start Alarm')
```

제가 파이썬을 하도 안해서 기억이 안나는데 이런 식으로 하면 좀 더 깔끔하지 않을까요?